### DISCUSSION TITLE
Partial width glide typing bug

### DISCUSSION THREAD

**Discussion by VA1DER** [florisboard/florisboard#1839] [Q&A]
I'm using 0.3.12 because it's the last version to support word suggestions.  I've noticed a bug where when I set a custom width for one handed mode that glide typing breaks.  It looks like the glide typing parser doesn't  take onto account the letter positions on the scaled keyboard.  It's trying to parse the glide typing as if the keyboard were full sized.

I'm just curious if this bug is already fixed in recent versions?

**Comment by patrickgold** [comment]
Yes, in the current release this is fixed, one-handed mode is taken into account.
